ethereum/core/src/main/java/org/hyperledger/besu/ethereum/privacy/PrivateWorldStateReader.java

```diff
@@ -46,4 +54,20 @@ public PrivateWorldStateReader(
         .flatMap(worldState -> Optional.ofNullable(worldState.get(contractAddress)))
         .flatMap(account -> Optional.ofNullable(account.getCode()));
   }
+
+  public List<PrivateTransactionMetadata> getPrivateTransactionsMetadata(
```

Comment:
should that be "getPrivateTransactionMetadataList"?

Reply:
Done